ERROR - 2024-11-01 06:00:21 --> mysqli_sql_exception: Unknown column 'updated_at' in 'field list' in /Users/galihagung/Documents/Development/ulilalbab/dev-psb.ulilalbabkra.sch.id/psb-ulilalbab/vendor/codeigniter4/framework/system/Database/MySQLi/Connection.php:312
Stack trace:
#0 /Users/galihagung/Documents/Development/ulilalbab/dev-psb.ulilalbabkra.sch.id/psb-ulilalbab/vendor/codeigniter4/framework/system/Database/MySQLi/Connection.php(312): mysqli->query('INSERT INTO `fi...', 0)
#1 /Users/galihagung/Documents/Development/ulilalbab/dev-psb.ulilalbabkra.sch.id/psb-ulilalbab/vendor/codeigniter4/framework/system/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `fi...')
#2 /Users/galihagung/Documents/Development/ulilalbab/dev-psb.ulilalbabkra.sch.id/psb-ulilalbab/vendor/codeigniter4/framework/system/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `fi...')
#3 /Users/galihagung/Documents/Development/ulilalbab/dev-psb.ulilalbabkra.sch.id/psb-ulilalbab/vendor/codeigniter4/framework/system/Database/BaseBuilder.php(2332): CodeIgniter\Database\BaseConnection->query('INSERT INTO `fi...', Array, false)
#4 /Users/galihagung/Documents/Development/ulilalbab/dev-psb.ulilalbabkra.sch.id/psb-ulilalbab/vendor/codeigniter4/framework/system/Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 /Users/galihagung/Documents/Development/ulilalbab/dev-psb.ulilalbabkra.sch.id/psb-ulilalbab/vendor/codeigniter4/framework/system/BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 /Users/galihagung/Documents/Development/ulilalbab/dev-psb.ulilalbabkra.sch.id/psb-ulilalbab/vendor/codeigniter4/framework/system/Model.php(802): CodeIgniter\BaseModel->insert(Array, true)
#7 /Users/galihagung/Documents/Development/ulilalbab/dev-psb.ulilalbabkra.sch.id/psb-ulilalbab/app/Controllers/Uploads.php(54): CodeIgniter\Model->insert(Array)
#8 /Users/galihagung/Documents/Development/ulilalbab/dev-psb.ulilalbabkra.sch.id/psb-ulilalbab/vendor/codeigniter4/framework/system/CodeIgniter.php(933): App\Controllers\Uploads->uploadFoto()
#9 /Users/galihagung/Documents/Development/ulilalbab/dev-psb.ulilalbabkra.sch.id/psb-ulilalbab/vendor/codeigniter4/framework/system/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Uploads))
#10 /Users/galihagung/Documents/Development/ulilalbab/dev-psb.ulilalbabkra.sch.id/psb-ulilalbab/vendor/codeigniter4/framework/system/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 /Users/galihagung/Documents/Development/ulilalbab/dev-psb.ulilalbabkra.sch.id/psb-ulilalbab/vendor/codeigniter4/framework/system/Boot.php(325): CodeIgniter\CodeIgniter->run()
#12 /Users/galihagung/Documents/Development/ulilalbab/dev-psb.ulilalbabkra.sch.id/psb-ulilalbab/vendor/codeigniter4/framework/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 /Users/galihagung/Documents/Development/ulilalbab/dev-psb.ulilalbabkra.sch.id/psb-ulilalbab/public/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 /Users/galihagung/Documents/Development/ulilalbab/dev-psb.ulilalbabkra.sch.id/psb-ulilalbab/vendor/codeigniter4/framework/system/rewrite.php(44): require_once('/Users/galihagu...')
#15 {main}
CRITICAL - 2024-11-01 06:00:21 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'updated_at' in 'field list'
[Method: POST, Route: berkas/upload/foto]
in SYSTEMPATH/Database/BaseConnection.php on line 688.
 1 SYSTEMPATH/Database/BaseBuilder.php(2332): CodeIgniter\Database\BaseConnection->query('INSERT INTO `files` (`jenis`, `path`, `id_siswa`, `created_at`, `updated_at`) VALUES (:jenis:, :path:, :id_siswa:, :created_at:, :updated_at:)', [...], false)
 2 SYSTEMPATH/Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH/BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH/Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH/Controllers/Uploads.php(54): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Uploads->uploadFoto()
 7 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Uploads))
 8 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH/rewrite.php(44): require_once('/Users/galihagung/Documents/Development/ulilalbab/dev-psb.ulilalbabkra.sch.id/psb-ulilalbab/public/index.php')
CRITICAL - 2024-11-01 06:00:21 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'updated_at' in 'field list'
in SYSTEMPATH/Database/MySQLi/Connection.php on line 317.
 1 SYSTEMPATH/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `files` (`jenis`, `path`, `id_siswa`, `created_at`, `updated_at`) VALUES (\'foto\', \'coba-coba\', \'60\', \'2024-11-01 06:00:21\', \'2024-11-01 06:00:21\')')
 2 SYSTEMPATH/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `files` (`jenis`, `path`, `id_siswa`, `created_at`, `updated_at`) VALUES (\'foto\', \'coba-coba\', \'60\', \'2024-11-01 06:00:21\', \'2024-11-01 06:00:21\')')
 3 SYSTEMPATH/Database/BaseBuilder.php(2332): CodeIgniter\Database\BaseConnection->query('INSERT INTO `files` (`jenis`, `path`, `id_siswa`, `created_at`, `updated_at`) VALUES (:jenis:, :path:, :id_siswa:, :created_at:, :updated_at:)', [...], false)
 4 SYSTEMPATH/Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH/BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH/Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH/Controllers/Uploads.php(54): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Uploads->uploadFoto()
 9 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Uploads))
10 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
14 SYSTEMPATH/rewrite.php(44): require_once('/Users/galihagung/Documents/Development/ulilalbab/dev-psb.ulilalbabkra.sch.id/psb-ulilalbab/public/index.php')
CRITICAL - 2024-11-01 06:00:21 --> [Caused by] mysqli_sql_exception: Unknown column 'updated_at' in 'field list'
in SYSTEMPATH/Database/MySQLi/Connection.php on line 312.
 1 SYSTEMPATH/Database/MySQLi/Connection.php(312): mysqli->query('INSERT INTO `files` (`jenis`, `path`, `id_siswa`, `created_at`, `updated_at`) VALUES (\'foto\', \'coba-coba\', \'60\', \'2024-11-01 06:00:21\', \'2024-11-01 06:00:21\')', 0)
 2 SYSTEMPATH/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `files` (`jenis`, `path`, `id_siswa`, `created_at`, `updated_at`) VALUES (\'foto\', \'coba-coba\', \'60\', \'2024-11-01 06:00:21\', \'2024-11-01 06:00:21\')')
 3 SYSTEMPATH/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `files` (`jenis`, `path`, `id_siswa`, `created_at`, `updated_at`) VALUES (\'foto\', \'coba-coba\', \'60\', \'2024-11-01 06:00:21\', \'2024-11-01 06:00:21\')')
 4 SYSTEMPATH/Database/BaseBuilder.php(2332): CodeIgniter\Database\BaseConnection->query('INSERT INTO `files` (`jenis`, `path`, `id_siswa`, `created_at`, `updated_at`) VALUES (:jenis:, :path:, :id_siswa:, :created_at:, :updated_at:)', [...], false)
 5 SYSTEMPATH/Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH/BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH/Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH/Controllers/Uploads.php(54): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Uploads->uploadFoto()
10 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Uploads))
11 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
15 SYSTEMPATH/rewrite.php(44): require_once('/Users/galihagung/Documents/Development/ulilalbab/dev-psb.ulilalbabkra.sch.id/psb-ulilalbab/public/index.php')
ERROR - 2024-11-01 06:01:52 --> mysqli_sql_exception: Unknown column 'updated_at' in 'field list' in /Users/galihagung/Documents/Development/ulilalbab/dev-psb.ulilalbabkra.sch.id/psb-ulilalbab/vendor/codeigniter4/framework/system/Database/MySQLi/Connection.php:312
Stack trace:
#0 /Users/galihagung/Documents/Development/ulilalbab/dev-psb.ulilalbabkra.sch.id/psb-ulilalbab/vendor/codeigniter4/framework/system/Database/MySQLi/Connection.php(312): mysqli->query('INSERT INTO `fi...', 0)
#1 /Users/galihagung/Documents/Development/ulilalbab/dev-psb.ulilalbabkra.sch.id/psb-ulilalbab/vendor/codeigniter4/framework/system/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `fi...')
#2 /Users/galihagung/Documents/Development/ulilalbab/dev-psb.ulilalbabkra.sch.id/psb-ulilalbab/vendor/codeigniter4/framework/system/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `fi...')
#3 /Users/galihagung/Documents/Development/ulilalbab/dev-psb.ulilalbabkra.sch.id/psb-ulilalbab/vendor/codeigniter4/framework/system/Database/BaseBuilder.php(2332): CodeIgniter\Database\BaseConnection->query('INSERT INTO `fi...', Array, false)
#4 /Users/galihagung/Documents/Development/ulilalbab/dev-psb.ulilalbabkra.sch.id/psb-ulilalbab/vendor/codeigniter4/framework/system/Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 /Users/galihagung/Documents/Development/ulilalbab/dev-psb.ulilalbabkra.sch.id/psb-ulilalbab/vendor/codeigniter4/framework/system/BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 /Users/galihagung/Documents/Development/ulilalbab/dev-psb.ulilalbabkra.sch.id/psb-ulilalbab/vendor/codeigniter4/framework/system/Model.php(802): CodeIgniter\BaseModel->insert(Array, true)
#7 /Users/galihagung/Documents/Development/ulilalbab/dev-psb.ulilalbabkra.sch.id/psb-ulilalbab/app/Controllers/Uploads.php(54): CodeIgniter\Model->insert(Array)
#8 /Users/galihagung/Documents/Development/ulilalbab/dev-psb.ulilalbabkra.sch.id/psb-ulilalbab/vendor/codeigniter4/framework/system/CodeIgniter.php(933): App\Controllers\Uploads->uploadFoto()
#9 /Users/galihagung/Documents/Development/ulilalbab/dev-psb.ulilalbabkra.sch.id/psb-ulilalbab/vendor/codeigniter4/framework/system/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Uploads))
#10 /Users/galihagung/Documents/Development/ulilalbab/dev-psb.ulilalbabkra.sch.id/psb-ulilalbab/vendor/codeigniter4/framework/system/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 /Users/galihagung/Documents/Development/ulilalbab/dev-psb.ulilalbabkra.sch.id/psb-ulilalbab/vendor/codeigniter4/framework/system/Boot.php(325): CodeIgniter\CodeIgniter->run()
#12 /Users/galihagung/Documents/Development/ulilalbab/dev-psb.ulilalbabkra.sch.id/psb-ulilalbab/vendor/codeigniter4/framework/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 /Users/galihagung/Documents/Development/ulilalbab/dev-psb.ulilalbabkra.sch.id/psb-ulilalbab/public/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 /Users/galihagung/Documents/Development/ulilalbab/dev-psb.ulilalbabkra.sch.id/psb-ulilalbab/vendor/codeigniter4/framework/system/rewrite.php(44): require_once('/Users/galihagu...')
#15 {main}
CRITICAL - 2024-11-01 06:01:52 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'updated_at' in 'field list'
[Method: POST, Route: berkas/upload/foto]
in SYSTEMPATH/Database/BaseConnection.php on line 688.
 1 SYSTEMPATH/Database/BaseBuilder.php(2332): CodeIgniter\Database\BaseConnection->query('INSERT INTO `files` (`jenis`, `path`, `id_siswa`, `created_at`, `updated_at`) VALUES (:jenis:, :path:, :id_siswa:, :created_at:, :updated_at:)', [...], false)
 2 SYSTEMPATH/Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH/BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH/Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH/Controllers/Uploads.php(54): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Uploads->uploadFoto()
 7 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Uploads))
 8 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH/rewrite.php(44): require_once('/Users/galihagung/Documents/Development/ulilalbab/dev-psb.ulilalbabkra.sch.id/psb-ulilalbab/public/index.php')
CRITICAL - 2024-11-01 06:01:52 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'updated_at' in 'field list'
in SYSTEMPATH/Database/MySQLi/Connection.php on line 317.
 1 SYSTEMPATH/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `files` (`jenis`, `path`, `id_siswa`, `created_at`, `updated_at`) VALUES (\'foto\', \'coba-coba\', \'60\', \'2024-11-01 06:01:51\', \'2024-11-01 06:01:51\')')
 2 SYSTEMPATH/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `files` (`jenis`, `path`, `id_siswa`, `created_at`, `updated_at`) VALUES (\'foto\', \'coba-coba\', \'60\', \'2024-11-01 06:01:51\', \'2024-11-01 06:01:51\')')
 3 SYSTEMPATH/Database/BaseBuilder.php(2332): CodeIgniter\Database\BaseConnection->query('INSERT INTO `files` (`jenis`, `path`, `id_siswa`, `created_at`, `updated_at`) VALUES (:jenis:, :path:, :id_siswa:, :created_at:, :updated_at:)', [...], false)
 4 SYSTEMPATH/Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH/BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH/Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH/Controllers/Uploads.php(54): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Uploads->uploadFoto()
 9 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Uploads))
10 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
14 SYSTEMPATH/rewrite.php(44): require_once('/Users/galihagung/Documents/Development/ulilalbab/dev-psb.ulilalbabkra.sch.id/psb-ulilalbab/public/index.php')
CRITICAL - 2024-11-01 06:01:52 --> [Caused by] mysqli_sql_exception: Unknown column 'updated_at' in 'field list'
in SYSTEMPATH/Database/MySQLi/Connection.php on line 312.
 1 SYSTEMPATH/Database/MySQLi/Connection.php(312): mysqli->query('INSERT INTO `files` (`jenis`, `path`, `id_siswa`, `created_at`, `updated_at`) VALUES (\'foto\', \'coba-coba\', \'60\', \'2024-11-01 06:01:51\', \'2024-11-01 06:01:51\')', 0)
 2 SYSTEMPATH/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `files` (`jenis`, `path`, `id_siswa`, `created_at`, `updated_at`) VALUES (\'foto\', \'coba-coba\', \'60\', \'2024-11-01 06:01:51\', \'2024-11-01 06:01:51\')')
 3 SYSTEMPATH/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `files` (`jenis`, `path`, `id_siswa`, `created_at`, `updated_at`) VALUES (\'foto\', \'coba-coba\', \'60\', \'2024-11-01 06:01:51\', \'2024-11-01 06:01:51\')')
 4 SYSTEMPATH/Database/BaseBuilder.php(2332): CodeIgniter\Database\BaseConnection->query('INSERT INTO `files` (`jenis`, `path`, `id_siswa`, `created_at`, `updated_at`) VALUES (:jenis:, :path:, :id_siswa:, :created_at:, :updated_at:)', [...], false)
 5 SYSTEMPATH/Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH/BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH/Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH/Controllers/Uploads.php(54): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Uploads->uploadFoto()
10 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Uploads))
11 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
15 SYSTEMPATH/rewrite.php(44): require_once('/Users/galihagung/Documents/Development/ulilalbab/dev-psb.ulilalbabkra.sch.id/psb-ulilalbab/public/index.php')
ERROR - 2024-11-01 06:05:36 --> mysqli_sql_exception: Unknown column 'updated_at' in 'field list' in /Users/galihagung/Documents/Development/ulilalbab/dev-psb.ulilalbabkra.sch.id/psb-ulilalbab/vendor/codeigniter4/framework/system/Database/MySQLi/Connection.php:312
Stack trace:
#0 /Users/galihagung/Documents/Development/ulilalbab/dev-psb.ulilalbabkra.sch.id/psb-ulilalbab/vendor/codeigniter4/framework/system/Database/MySQLi/Connection.php(312): mysqli->query('INSERT INTO `fi...', 0)
#1 /Users/galihagung/Documents/Development/ulilalbab/dev-psb.ulilalbabkra.sch.id/psb-ulilalbab/vendor/codeigniter4/framework/system/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `fi...')
#2 /Users/galihagung/Documents/Development/ulilalbab/dev-psb.ulilalbabkra.sch.id/psb-ulilalbab/vendor/codeigniter4/framework/system/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `fi...')
#3 /Users/galihagung/Documents/Development/ulilalbab/dev-psb.ulilalbabkra.sch.id/psb-ulilalbab/vendor/codeigniter4/framework/system/Database/BaseBuilder.php(2332): CodeIgniter\Database\BaseConnection->query('INSERT INTO `fi...', Array, false)
#4 /Users/galihagung/Documents/Development/ulilalbab/dev-psb.ulilalbabkra.sch.id/psb-ulilalbab/vendor/codeigniter4/framework/system/Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 /Users/galihagung/Documents/Development/ulilalbab/dev-psb.ulilalbabkra.sch.id/psb-ulilalbab/vendor/codeigniter4/framework/system/BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 /Users/galihagung/Documents/Development/ulilalbab/dev-psb.ulilalbabkra.sch.id/psb-ulilalbab/vendor/codeigniter4/framework/system/Model.php(802): CodeIgniter\BaseModel->insert(Array, true)
#7 /Users/galihagung/Documents/Development/ulilalbab/dev-psb.ulilalbabkra.sch.id/psb-ulilalbab/app/Controllers/Uploads.php(54): CodeIgniter\Model->insert(Array)
#8 /Users/galihagung/Documents/Development/ulilalbab/dev-psb.ulilalbabkra.sch.id/psb-ulilalbab/vendor/codeigniter4/framework/system/CodeIgniter.php(933): App\Controllers\Uploads->uploadFoto()
#9 /Users/galihagung/Documents/Development/ulilalbab/dev-psb.ulilalbabkra.sch.id/psb-ulilalbab/vendor/codeigniter4/framework/system/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Uploads))
#10 /Users/galihagung/Documents/Development/ulilalbab/dev-psb.ulilalbabkra.sch.id/psb-ulilalbab/vendor/codeigniter4/framework/system/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 /Users/galihagung/Documents/Development/ulilalbab/dev-psb.ulilalbabkra.sch.id/psb-ulilalbab/vendor/codeigniter4/framework/system/Boot.php(325): CodeIgniter\CodeIgniter->run()
#12 /Users/galihagung/Documents/Development/ulilalbab/dev-psb.ulilalbabkra.sch.id/psb-ulilalbab/vendor/codeigniter4/framework/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 /Users/galihagung/Documents/Development/ulilalbab/dev-psb.ulilalbabkra.sch.id/psb-ulilalbab/public/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 /Users/galihagung/Documents/Development/ulilalbab/dev-psb.ulilalbabkra.sch.id/psb-ulilalbab/vendor/codeigniter4/framework/system/rewrite.php(44): require_once('/Users/galihagu...')
#15 {main}
CRITICAL - 2024-11-01 06:05:36 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'updated_at' in 'field list'
[Method: POST, Route: berkas/upload/foto]
in SYSTEMPATH/Database/BaseConnection.php on line 688.
 1 SYSTEMPATH/Database/BaseBuilder.php(2332): CodeIgniter\Database\BaseConnection->query('INSERT INTO `files` (`jenis`, `path`, `id_siswa`, `created_at`, `updated_at`) VALUES (:jenis:, :path:, :id_siswa:, :created_at:, :updated_at:)', [...], false)
 2 SYSTEMPATH/Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH/BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH/Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH/Controllers/Uploads.php(54): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Uploads->uploadFoto()
 7 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Uploads))
 8 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH/rewrite.php(44): require_once('/Users/galihagung/Documents/Development/ulilalbab/dev-psb.ulilalbabkra.sch.id/psb-ulilalbab/public/index.php')
CRITICAL - 2024-11-01 06:05:36 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'updated_at' in 'field list'
in SYSTEMPATH/Database/MySQLi/Connection.php on line 317.
 1 SYSTEMPATH/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `files` (`jenis`, `path`, `id_siswa`, `created_at`, `updated_at`) VALUES (\'foto\', \'coba-coba\', \'60\', \'2024-11-01 06:05:36\', \'2024-11-01 06:05:36\')')
 2 SYSTEMPATH/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `files` (`jenis`, `path`, `id_siswa`, `created_at`, `updated_at`) VALUES (\'foto\', \'coba-coba\', \'60\', \'2024-11-01 06:05:36\', \'2024-11-01 06:05:36\')')
 3 SYSTEMPATH/Database/BaseBuilder.php(2332): CodeIgniter\Database\BaseConnection->query('INSERT INTO `files` (`jenis`, `path`, `id_siswa`, `created_at`, `updated_at`) VALUES (:jenis:, :path:, :id_siswa:, :created_at:, :updated_at:)', [...], false)
 4 SYSTEMPATH/Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH/BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH/Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH/Controllers/Uploads.php(54): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Uploads->uploadFoto()
 9 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Uploads))
10 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
14 SYSTEMPATH/rewrite.php(44): require_once('/Users/galihagung/Documents/Development/ulilalbab/dev-psb.ulilalbabkra.sch.id/psb-ulilalbab/public/index.php')
CRITICAL - 2024-11-01 06:05:36 --> [Caused by] mysqli_sql_exception: Unknown column 'updated_at' in 'field list'
in SYSTEMPATH/Database/MySQLi/Connection.php on line 312.
 1 SYSTEMPATH/Database/MySQLi/Connection.php(312): mysqli->query('INSERT INTO `files` (`jenis`, `path`, `id_siswa`, `created_at`, `updated_at`) VALUES (\'foto\', \'coba-coba\', \'60\', \'2024-11-01 06:05:36\', \'2024-11-01 06:05:36\')', 0)
 2 SYSTEMPATH/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `files` (`jenis`, `path`, `id_siswa`, `created_at`, `updated_at`) VALUES (\'foto\', \'coba-coba\', \'60\', \'2024-11-01 06:05:36\', \'2024-11-01 06:05:36\')')
 3 SYSTEMPATH/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `files` (`jenis`, `path`, `id_siswa`, `created_at`, `updated_at`) VALUES (\'foto\', \'coba-coba\', \'60\', \'2024-11-01 06:05:36\', \'2024-11-01 06:05:36\')')
 4 SYSTEMPATH/Database/BaseBuilder.php(2332): CodeIgniter\Database\BaseConnection->query('INSERT INTO `files` (`jenis`, `path`, `id_siswa`, `created_at`, `updated_at`) VALUES (:jenis:, :path:, :id_siswa:, :created_at:, :updated_at:)', [...], false)
 5 SYSTEMPATH/Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH/BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH/Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH/Controllers/Uploads.php(54): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Uploads->uploadFoto()
10 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Uploads))
11 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
15 SYSTEMPATH/rewrite.php(44): require_once('/Users/galihagung/Documents/Development/ulilalbab/dev-psb.ulilalbabkra.sch.id/psb-ulilalbab/public/index.php')
